vtex/faststore
# Change orderform toast to be more error like

## What's the purpose of this pull request?
As the title says.

## How it works? 
Add a new property `type` to Toast. Also, change the useToast from orderform to use this type property, also change the duration to 6s.

## How to test it?
1. Go to one o these stores: [storecomponents](https://github.com/vtex-sites/storecomponents.store/pull/730), [marinbrasil](https://github.com/vtex-sites/marinbrasil.store/pull/440) and [btg](https://github.com/vtex-sites/btglobal.store/pull/394)
2. Add 100 products to cause an orderform error
3. see the red toast
